README-dockerhub.md

```diff
@@ -76,7 +76,7 @@ There are some env vars for overriding this data;
 
 | Argument         | Purpose                                    | Example Usage                                 |
 | ---------------- | ------------------------------------------ | --------------------------------------------- |
-| `--debug`        | Get more detailed output as Trivy runs.    |                                               |
+| `--debug`        | Get more detailed output as Trivy runs.    |                                               | `--debug`
```

Comment:
Is this second --debug supposed to be under the example usage column? If so, shouldn't it then be 
`--debug` |